server/game/gamesteps/actionwindow.js

```diff
@@ -116,7 +116,18 @@ class ActionWindow extends UiPrompt {
     pass() {
         this.game.addMessage('{0} passes', this.currentPlayer);
 
-        if(this.prevPlayerPassed || !this.currentPlayer.opponent) {
+        let resolveConflictEarlyEffects = this.currentPlayer.getEffects(EffectNames.ResolveConflictEarly);
+        var resolveConflictEarly = false;
+        for(let i = 0; i < resolveConflictEarlyEffects.length; i++) {
```

Comment:
This is only going to countdown when a player passes.  I'm not sure the idea of decrementing a counter will work generally in this case, because I don't think you can change the value stored in the effect in this way.

Reply:
I was trying to allow specifying a number of turns left to make the effect a little more generic in case it's needed later. Would you recommend just checking for the effect itself and ending the conflict? Or should I move tracking the turns left to part of the player itself so I can modify it?

Reply:
I think if you want to do a counter, you probably need to track it in the action window itself. To be honest though, I think it's probably a case of overengineering, and you should just check for the effect and end the conflict.  If we get something which needs a tracker, we'll come up with a solution at that point.